CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ option(PYTHON "Enable Python bindings" OFF)
 option(CSHARP "Enable C# bindings" OFF)
 option(COVERAGE "Enable gcov coverage" OFF)
 option(EXAMPLES "Enable compilation of examples" ON)
+set(PYTHON_CONFIG "python-config" CACHE STRING "python-config for build env config")
```

Comment:
May need to make this configurable instead of hardcoded string. @colinpattison thoughts?

Reply:
As discussed, using 'set' creates a variable that can be overridden using -D e.g. -DPYTHON_CONFIG.